packages/flame/lib/src/game/mixins/has_fps.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:math';
+
+import 'package:collection/collection.dart';
+import 'package:flutter/scheduler.dart';
+
+import '../../../game.dart';
+
+const _maxFrames = 60.0;
+const frameInterval =
+    Duration(microseconds: Duration.microsecondsPerSecond ~/ _maxFrames);
+
+@Deprecated(
+  'This will be removed in favor of `HasFPS` in v1.3.0',
+)
+typedef FPSCounter = HasFPS;
+
+mixin HasFPS on Game {
+  List<FrameTiming> _previousTimings = [];
+
+  @override
+  void onTimingsCallback(List<FrameTiming> timings) =>
+      _previousTimings = timings;
+
+  /// Returns the FPS based on the frame times from [onTimingsCallback].
+  double fps([
+    @Deprecated('This will be removed in v1.3.0') int average = 1,
```

Comment:
That's a bit of a weird parameter, isn't it? It's not used in any way; it was not used in the old code either. Maybe we could just remove it immediately?

Reply:
People still use it, even we used it in some examples (because it did have effect back in the days). 😅
No harm in giving folks a heads up before removing it.

Reply:
Maybe we should add to the comment that it actually isn't doing anything?